qa/die-with-dignity/build.gradle

```diff
@@ -29,3 +29,8 @@ tasks.named("test").configure {
   enabled = false
 }
 
+tasks.named('splitPackagesAudit').configure {
```

Comment:
Should we even care given this is a qa project and not production code? IMO we shouldn't even apply this check to this project.

Reply:
This qa project creates a plugin. That plugin will need to operate in the same environment as server, so we will still need to ensure it doesn't split any server packages.

Reply:
Makes sense.